types/ltijs/lib/Provider/Provider.d.ts

```diff
@@ -36,14 +35,31 @@ export interface ProviderOptions {
         secure?: boolean | undefined;
         sameSite?: string | undefined;
```

Comment:
`cookies.domain` [seems to be missing](https://github.com/Cvmcosta/ltijs/blob/decf5366f90f711c8482154d46c105f4a154894a/src/Provider/Provider.js#L168)

Reply:
Thanks for spotting that one too, added.